James/quotient intro (#29)
quotient-ai/quotient-docs: promptlab/iq.mdx

---

> title: "Quotient IQ - Intelligent Prompt Optimization"

Reviewer: ```suggestion
title: "Quotient IQ - Your AI engineer copilot"
```

> description: "Learn how to use Quotient IQ to optimize and improve your prompts with intelligent feedback."

Reviewer: ```suggestion
description: "Learn how to use Quotient IQ for real-time, feedback-aligned prompt optimization."
```

---

## 🚀 Step 6: Run the Updated Prompt

Once you’ve accepted the changes, run the updated prompt again to observe how the AI's responses have improved. IQ ensures a continuous cycle of improvement by iterating on feedback.

Keep testing, refining, and adjusting your prompts to reach the best possible performance.

---

## 💾 Step 7: Save the Optimized Prompt

Once you're satisfied with the changes, save your optimized prompt. You can do this by clicking **Save new version**. This version is now stored for future use, ensuring that you have a well-optimized prompt ready for similar tasks.

<Image src="../assets/iq/save-updated-prompt.png" alt="Save Updated Prompt" />

---

## 🔍 Best Practices for Using IQ

Here are some best practices to follow when using IQ:

- **Be specific in your feedback**: The more specific your feedback, the better IQ can optimize your prompt.
- **Use variables creatively**: Variables allow you to create flexible prompts that adapt to various contexts, making them more reusable.
- **Test iteratively**: Keep running and refining your prompt based on real-world scenarios to achieve the most effective results.

---

> For further assistance, feel free to reach out to our our founders at [contact@quotientai.co](mailto:contact@quotientai.co) or check out more detailed documentation on [PromptLab](../promptlab/promptlab.mdx).

Reviewer 2: ```suggestion
For further assistance, feel free to reach out to our founders at [contact@quotientai.co](mailto:contact@quotientai.co) or check out more detailed documentation on [PromptLab](../promptlab/promptlab.mdx).
```
